src/os/ObjectStore.h

```diff
@@ -269,8 +269,8 @@ class ObjectStore {
    *   sobject_encoding detects an older/simpler version of oid
    *   present in pre-bobtail versions of ceph.  use_pool_override
    *   also detects a situation where the pool of an oid can be
-   *   override for legacy operations/buffers.  For non-legacy
-   *   implementation of ObjectStore, neither of these fields is
+   *   overriden for legacy operations/buffers.  For non-legacy
+   *   implementations of ObjectStore, neither of these fields are
```

Comment:
i am not sure if `neither` should be plural here. the verb *can* be singular here also. see http://www.bbc.co.uk/learningenglish/marathi/course/intermediate/unit-3/tab/grammar

> After neither (of) the verb is always positive. It can be singular or plural after neither of.
>
>> Neither of these answers is/are correct.